.github/workflows/playwright.yml

```
@@ -0,0 +1,43 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        cache: yarn
+
+    - name: Install dependencies
+      run: npm install -g yarn && yarn
```

I think `yarn` should be installed by the above step
```suggestion
    - name: Install dependencies
      run: yarn
```

---

@justwiebe thanks, I'll try that.